VMware.CloudFoundation.PasswordManagement.psm1

```diff
@@ -2265,9 +2332,24 @@ Function Update-SsoAccountLockout {
                     if (($vcfVcenterDetails = Get-vCenterServerDetail -server $server -user $user -pass $pass -domain $domain)) {
                         if (Test-SsoConnection -server $($vcfVcenterDetails.fqdn)) {
                             if (Test-SsoAuthentication -server $vcfVcenterDetails.fqdn -user $vcfVcenterDetails.ssoAdmin -pass $vcfVcenterDetails.ssoAdminPass) {
-                                if ((Get-SsoLockoutPolicy).MaxFailedAttempts -ne $failures -or (Get-SsoLockoutPolicy).FailedAttemptIntervalSec -ne $failureInterval -or (Get-SsoLockoutPolicy).AutoUnlockIntervalSec -ne $unlockInterval) {
-                                    Get-SsoLockoutPolicy | Set-SsoLockoutPolicy  -AutoUnlockIntervalSec $unlockInterval -FailedAttemptIntervalSec $failureInterval -MaxFailedAttempts $failures | Out-Null
-                                    if ((Get-SsoLockoutPolicy).MaxFailedAttempts -eq $failures -and (Get-SsoLockoutPolicy).FailedAttemptIntervalSec -eq $failureInterval -and (Get-SsoLockoutPolicy).AutoUnlockIntervalSec -eq $unlockInterval) {
+                                try {
+                                    $certificateValidator = New-Object 'VMware.vSphere.SsoAdmin.Utils.AcceptAllX509CertificateValidator'
+                                    $securePass = ConvertTo-SecureString $vcfVcenterDetails.ssoAdminPass -AsPlainText -Force
+                                    $ssoAdminServer = New-Object `
+                                    'VMware.vSphere.SsoAdminClient.DataTypes.SsoAdminServer' `
+                                    -ArgumentList @(
+                                        $vcfVcenterDetails.fqdn,
+                                        $vcfVcenterDetails.ssoAdmin,
+                                        $securePass,
+                                    $certificateValidator)
+                                } catch {
```

Comment:
```suggestion
                                } Catch {
```

Reply:
Resolved in https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-password-management/pull/30/commits/34c0aa64349c7ef45bcb9819bff4f616b46f73f2.